app/src/main/java/com/github/se/eventradar/ui/viewProfile/ProfileUi.kt

```diff
@@ -113,68 +161,183 @@ fun ProfileUi(
+                    label = { Text("Username") },
+                    isError = uiState.userNameIsError,
+                    colors =
+                        OutlinedTextFieldDefaults.colors()
+                            .copy(
+                                focusedPrefixColor = MaterialTheme.colorScheme.primary,
+                                unfocusedLabelColor = MaterialTheme.colorScheme.primary,
+                            ),
+                    supportingText = { Text("Your username must be unique") })
+                Column(modifier = Modifier.fillMaxWidth().testTag("leftAlignedViewProfileColumn")) {
+                  OutlinedTextField(
+                      value = uiState.bio,
+                      onValueChange = viewModel::onBioChanged,
+                      modifier =
+                          Modifier.fillMaxWidth().padding(top = 10.dp).testTag("bioTextField"),
+                      label = { Text("Bio") },
+                      isError = uiState.bioIsError,
+                      colors =
+                          OutlinedTextFieldDefaults.colors()
+                              .copy(
+                                  focusedPrefixColor = MaterialTheme.colorScheme.primary,
+                                  unfocusedLabelColor = MaterialTheme.colorScheme.primary,
+                              ))
+                  Row(
+                      horizontalArrangement = Arrangement.SpaceBetween,
+                      modifier = Modifier.padding(top = 10.dp).testTag("phoneNumberBirthDateRow")) {
+                        OutlinedTextField(
+                            value = uiState.phoneNumber,
+                            onValueChange = viewModel::onPhoneNumberChanged,
+                            modifier = Modifier.weight(1f).testTag("phoneNumberTextField"),
+                            label = { Text("Phone Number") },
+                            isError = uiState.phoneNumberIsError,
+                            colors =
+                                OutlinedTextFieldDefaults.colors()
+                                    .copy(
+                                        focusedPrefixColor = MaterialTheme.colorScheme.primary,
+                                        unfocusedLabelColor = MaterialTheme.colorScheme.primary,
+                                    ))
+                        Spacer(
+                            modifier = Modifier.width(10.dp).testTag("phoneNumberBirthDateSpacer"))
+                        OutlinedTextField(
+                            value = uiState.birthDate,
+                            onValueChange = viewModel::onBirthDateChanged,
+                            modifier = Modifier.weight(1f).testTag("birthDateTextField"),
+                            label = { Text("Birth Date") },
+                            isError = uiState.birthDateIsError,
+                            colors =
```

Comment:
Agree with Nik regarding DOB format, Vicky changed it to use "." not "/". This will need to be changed in the firebase to ensure validation is correct and no crashes when pulling data. 

Also keyboard should be numeric too for DOB.